buf.gen.yaml

```diff
@@ -19,7 +19,12 @@ plugins:
       - paths=source_relative
       - module=github.com/blndgs/model
 
-  - plugin: buf.build/community/stephenh-ts-proto:v1.167.9
+  # there does not seem to be a way to generate
+  # both typings and actual code in one pass
+  - plugin: buf.build/bufbuild/es:v1.10.0
+    opt: target=dts
+    out: gen/ts
```

Comment:
Ideally, we can leave this out as the (tsc) build process should generate the `.d.ts` for users of the library
but this is here in case we ever want to inspect the generated typing files or something that is shipped to users